lib/engine/game/g_1856.rb

```diff
@@ -76,12 +76,52 @@ class G1856 < Base
         1 => { 3 => 10, 4 => 8, 5 => 7, 6 => 6 },
       }.freeze
 
+      DESTINATIONS = {
```

Comment:
why isn't this a property on the corporation?

Reply:
I figured this is the clearer way to do it for 1856 to be able to support the alternate destinations optional rule. Making it part of the corporation in the config would, in the case of using alternate destinations, require unsetting the destinations in the map and the corporations before setting new ones. This way I can have the game pick one of the static hash maps based on the rules and load that into the game

Reply:
ok, but you can still hive it set the corporation destiantions, i wouldn't make the 1856 version be the defacto standard so